Neos.Flow/Classes/Security/AccountIdentifier.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
```

Comment:
nitpick: empty line can be removed

Reply:
The "RoleCollection"/"AccountRoles" is one of the not done parts. I might not completely agree that it's a requirement of a Account. But I see pros/cons for both. But in terms of that - should we introduce a simple Collection class to hold the Roles? To avoud returning a array

Reply:
Right, I only looked at the "TODO list" afterwards :blush:
IMO roles absolutely have to be part of the interface or we can't replace the `Account` in the security framework (everything is role based there).
And, yes, it should be value objects too, along the lines of https://gist.github.com/bwaidelich/8266c980ab8c2db817bd4425a817390a (untested)

(And, no, it didn't want to imply that _you_ have to take care of this all *g)

Reply:
Arh, good point about the security framework (which might be splitted at one point #evilplan ;) )